packages/loot-core/src/server/accounts/rules.ts

```diff
@@ -384,12 +384,12 @@ export class Condition {
         if (fieldValue === null) {
           return false;
         }
-        return fieldValue.indexOf(this.value) !== -1;
+        return `${fieldValue}`.indexOf(this.value) !== -1;
```

Comment:
A more explicit type casting would be clearer here

```suggestion
        return String(fieldValue).indexOf(this.value) !== -1;
```